client/packages/system/src/Asset/ImportCatalogueItem/UploadTab.tsx

```diff
@@ -279,7 +281,7 @@ export const AssetItemUploadTab: FC<ImportPanel & AssetItemUploadTabProps> = ({
         </Grid>
       ) : null}
       <Stack spacing={2}>
-        <Upload onUpload={csvImport} />
+        <UploadDragAndDrop onUpload={csvImport} />
```

Comment:
should we not... always use UploadFile? Can't see why it would make sense on android anywhere?

Reply:
Well we could, but some things are only available on central and thus shouldn't be on Android. But will make them all use UploadFile 😉

Reply:
True... though yeah, still think healthy to expect someone will connect a tablet as a client to oms central at some point 😅

Reply:
Guess if you do that, we don't really need to isMobile check in the drag and drop component anymore....